py-rse/correct.Rmd

```diff
@@ -117,8 +127,9 @@ Traceback (most recent call last):
 ValueError: too many values to unpack (expected 2)
 ```
 
-A precondition (with informative error message) in the `main` function of `collate.py`
-requiring the input files to end in ".csv" can clear up any potential confusion.
+An assertion in the `main` function of `collate.py`
```

Comment:
We are going to need to think of a different example for using an assertion in our code. The check for `.csv` input files in `collate.py` is now done in the error handling chapter - see [this comment](https://github.com/merely-useful/merely-useful.github.io/pull/400#pullrequestreview-434469966) in #400 for details.

Reply:
Replaced example.